src/app/features/test-records/components/vehicle-header/vehicle-header.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="test_result_header">
+  <div class="test_name_and_date">
+    <h1 class="govuk-heading-l govuk-!-margin-bottom-2">{{ test?.testTypeId || '' | testTypeName: (selectAllTestTypes$ | async) }}</h1>
+    <div class="date_and_result">
+      <span class="govuk-caption-l">{{ testResult?.createdAt | date: 'dd MMM yyyy' }}</span>
```

Comment:
I assume we have dates in other parts of the app too and that we use the same format for all. Then a shared custom directive here would be helpful, like `govDate`.

Reply:
Unfortunately we use different formats all around so I don't see much benefit.